show/aliases.ini

```diff
@@ -3,3 +3,4 @@ running-configuration=runningconfiguration
 running-config=runningconfiguration
 startup-configuration=startupconfiguration
 startup-config=startupconfiguration
+reboot-cause=reboot cause
```

Comment:
Is this tested? I haven't attempted to alias a command to a command + subcommand before.

Reply:
good catch! need to be removed.

Reply:
So it didn't work?

Reply:
no need and it didn't work.

Reply:
No need, but it could have cleaned up the command syntax a bit. Good to know it doesn't work for the future. If we ever need this type of aliasing, we will have to do more work.